src/TimeseriesPrediction.jl

```diff
@@ -6,9 +6,16 @@ module TimeseriesPrediction
 using Reexport
 @reexport using DynamicalSystemsBase
 
+using Statistics, LinearAlgebra
+
 include("localmodeling.jl")
-include("streconstruction.jl")
-include("st_prediction.jl")
-include("spatiotemporalsystem.jl")
+include("reconstruction.jl")
 
+include("pca/pca.jl")
```

Comment:
Good question.
The functionality that I actually use from it is very limited. (mostly just `pcacov` + PCA struct)
It may not be worth separating it into a new package.

Reply:
Deleted a whole bunch of unnecessary stuff for the folder.
`fit` just stayed because I use it inside a test. (Maybe better to use the `PrincipalComponentAnalysis` for that and remove it from `TSP.jl` as well)

Do we even want to keep the `PCA` struct ?

Reply:
I removed the rest of the pca functions that we don't really need and made tests use
MultivariateStats for comparison.
BTW `MultivariateStats` has been updated to v1.0 and we're now back to using
an exact copy of their code.
Unless we want to further optimize/customize our PCA struct, we can also go back
to using `MultivariateStats`.
( Only _issue_, I call `pcacov` which is not exported... so no guarantees on their side)